projects/packages/my-jetpack/_inc/components/product-interstitial/ProductInterstitial.jsx

```diff
@@ -0,0 +1,304 @@
+	);
+
+	const navigateToMyJetpackOverviewPage = useMyJetpackNavigate( MyJetpackRoutes.Home );
+
+	const clickHandler = useCallback(
+		( checkout, product, tier ) => {
+			ctaCallback?.( { slug, product, tier } );
+
+			if ( product?.isBundle || directCheckout ) {
+				// Get straight to the checkout page.
+				checkout?.();
+				return;
+			}
+
+			activate(
+				{ productId: slug },
+				{
+					onSettled: activatedProduct => {
+						let postCheckoutUrl = activatedProduct?.post_checkout_url || myJetpackCheckoutUri;
+
+						// If the interstitial is highlighting a specific feature, use the post checkout URL for that feature, if available.
+						if ( feature && activatedProduct?.post_checkout_urls_by_feature?.[ feature ] ) {
+							postCheckoutUrl = activatedProduct.post_checkout_urls_by_feature[ feature ];
+						}
+
+						// there is a separate hasRequiredTier, but it is not implemented
+						const hasPaidPlanForProduct = product?.hasPaidPlanForProduct;
+						const isFree = tier
+							? product?.pricingForUi?.tiers?.[ tier ]?.isFree
+							: product?.pricingForUi?.isFree;
+						const isUpgradeToHigherTier =
+							tier && product?.pricingForUi?.tiers?.[ tier ] && ! isFree && product?.isUpgradable;
+						const needsPurchase = ( ! isFree && ! hasPaidPlanForProduct ) || isUpgradeToHigherTier;
+
+						// If the product is CRM, redirect the user to the Jetpack CRM pricing page.
+						// This is done because CRM is not part of the WP billing system
+						// and we can't send them to checkout like we can with the rest of the products
+						if ( product.pluginSlug === 'zero-bs-crm' && ! hasPaidPlanForProduct ) {
+							window.location.href = 'https://jetpackcrm.com/pricing/';
+							return;
+						}
+
+						// If no purchase is needed, redirect the user to the product screen.
+						if ( ! needsPurchase ) {
+							// for free products, we still initiate the site connection
+							handleRegisterSite().then( postRegisterRedirectUri => {
+								if ( ! postRegisterRedirectUri ) {
+									// Fall back to the My Jetpack overview page.
+									return navigateToMyJetpackOverviewPage();
+								}
+							} );
+
+							return;
+						}
+
+						// Redirect to the checkout page.
+						checkout?.( null, postCheckoutUrl );
+					},
+				}
+			);
+		},
+		[
+			myJetpackCheckoutUri,
+			feature,
+			ctaCallback,
+			slug,
+			directCheckout,
+			activate,
+			handleRegisterSite,
+			navigateToMyJetpackOverviewPage,
+		]
+	);
+
+	return (
+		<AdminPage
+			showHeader={ false }
+			showBackground={ false }
+			useInternalLinks={ shouldUseInternalLinks() }
+		>
+			<Container horizontalSpacing={ 3 } horizontalGap={ 3 }>
+				<Col className={ styles[ 'product-interstitial__header' ] }>
+					<GoBackLink onClick={ onClickGoBack } />
+					{ existingLicenseKeyUrl && (
+						<Text variant="body-small">
+							{ createInterpolateElement(
+								__(
+									'Already have an existing plan or license key? <a>Get started</a>.',
```

Comment:
Let us use the [existing string](https://github.com/search?q=repo%3AAutomattic%2Fjetpack%20%22Already%20have%20an%20existing%20plan%20or%20license%20key%3F%20%3Ca%3EClick%20here%20to%20get%20started%3C%2Fa%3E%22&type=code) which should already be translated

```suggestion
									'Already have an existing plan or license key? <a>Click here to get started</a>.',
```

Reply:
Done